backend/Bindings/Edna.Bindings.LtiAdvantage/Services/KeyVaultJwkProvider.cs

```diff
@@ -18,8 +20,8 @@ public async Task<JsonWebKey> GetJwk(string keyVaultIdentifier)
         {
             AzureServiceTokenProvider azureServiceTokenProvider = new AzureServiceTokenProvider();
             KeyVaultSecurityKey.AuthenticationCallback keyVaultAuthCallback = new KeyVaultSecurityKey.AuthenticationCallback(azureServiceTokenProvider.KeyVaultTokenCallback);
-
-            KeyVaultClient client = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(keyVaultAuthCallback));
+            EdnaExternalHttpHandler ednaExternalHttpHandler = new EdnaExternalHttpHandler();
```

Comment:
What's this change helping out with?

Reply:
Essentially passing the custom `HttpClient` handler as a parameter to the `KeyVaultClient`, so that it uses this instead of the default `HttpClient`.